ngrinder-core/build.gradle

```diff
@@ -16,10 +17,19 @@ dependencies {
     compile (group: "org.apache.commons", name: "commons-compress", version:"1.4.1")
     compile (group: "com.beust", name: "jcommander", version:"1.32")
     compile (group: "junit", name: "junit-dep", version:"4.8.2")
+    compile (group: "dnsjava", name: "dnsjava", version:"2.1.6")
+    compile (group: "asm", name: "asm", version:"3.3.1")
+    compile (group: "ch.qos.logback", name: "logback-classic", version:"1.0.0")
+    compile (group: "org.slf4j", name: "slf4j-api", version: slf4j_version)
+    compile (group: "net.sf.grinder", name: "grinder", version:"3.9.1") {
+        exclude (module: "clojure")
+        exclude (module: "jython")
+        exclude (module: "grinder-xmlbeans")
+        exclude (module: "jedit-syntax")
+    }
```

Comment:
Is this is really necessary? These are imported transparently by ngrinder-runtime.